libparsec/crates/client/tests/unit/workspace/remove_entry.rs

```diff
@@ -288,3 +293,171 @@ async fn remove_folder_with_local_changes(
         EntryStat::Folder { need_sync, .. } if !need_sync
     );
 }
+
+#[parsec_test(testbed = "minimal_client_ready", with_server)]
+async fn remove_confined_entry(
```

Comment:
We are testing the transaction, I would say to move the test to `libparsec/crates/client/tests/unit/workspace/folder_transactions.rs`.

That way to could use the helper defined here like `bootstrap_env`